membership_file/admin.py

```diff
@@ -214,7 +220,7 @@ class MemberYearAdmin(ExportActionMixin, admin.ModelAdmin):
     ##############################
     #  Export functionality
     resource_class = MembersFinancialResource
-    formats = (CSV,)
+    formats = (CSV,TSVUnicodeBOM,ODS,XLSX,)
```

Comment:
I don't know if the admin orders these types, but in case not. Please keep the order the same as defined in the MemberWithLog admin on line 58

Reply:
Oh, that was an oversight, sorry.